### DISCUSSION TITLE
Impossible to get approval

### DISCUSSION THREAD

**Discussion by johanguse** [community/community#128893] [GitHub Education]
### Hi everyone,

Hey hello!

I just approval from Jetbrain, but I am not sure about Github. I've used the same documentation in both. I keep submitting the photos, and I'm not sure if They will be approved or not. Is it possible to talk with someone real from Github? Maybe send by email all the documentation?

![image](https://github.com/community/community/assets/6184866/a41aaea0-ab13-4285-af45-63e595fa6bc2)

**Comment by barthelemyb** [comment]
I am having the same issue, every time I try to submit my proof it just takes me back to the upload approval page
